demisto_sdk/commands/doc_reviewer/doc_reviewer.py

```diff
@@ -302,7 +311,8 @@ def remove_punctuation(word):
     def check_word(self, word):
         """Check if a word is legal"""
         # check camel cases
-        if not self.no_camel_case and self.is_camel_case(word):
+        if not self.no_camel_case and self.is_camel_case(word) and not self.is_upper_case_word_plural(self.remove_punctuation(word)):
```

Comment:
what if `self.no_camel_case` is True? we don't want to call `is_upper_case_word_plural()` in this case?

Reply:
The function `is_upper_case_word_plural()` should be called in a case where the function `is_camel_case(word)` deems a word as camel case, but in fact it is not (like IPs, URLs, etc). Maybe it is better to move the logic in the function `is_upper_case_word_plural()` to `is_camel_case(word)`